bool pub_pc
bool terminate_flag
---
sensor_msgs/PointCloud2 octo_change
geometry_msgs/PoseStamped curr_pose